src/components/GiniIndex.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const GiniIndex = ({ giniIndex }) => (giniIndex && <div className="stats-no-box">
```

Comment:
Would be good to move `giniIndex` truthiness check even before calling the component